src/components/MermaidDash/MermaidDash.jsx

```diff
@@ -46,10 +49,17 @@ const MermaidDash = () => {
 
   const fetchData = useCallback(
     async (token = '') => {
+      if (isApiDataDoneLoading) {
+        return
+      }
       try {
+        setLoadedProjectCount(0)
+        setTotalProjectsCount(0)
```

Comment:
See related reply below. Since we are _summing_ the counts for all the calls to get chunks of data, we need to reinitialize it to 0 when we start a series of calls again, so we arent adding to numbers from previous calls.